octodns/__init__.py

```diff
@@ -1,6 +1,25 @@
-'OctoDNS: DNS as code - Tools for managing DNS across multiple providers'
-
 from __future__ import absolute_import, division, print_function, \
     unicode_literals
+import pkg_resources
+from os import path
+from setuptools.config import read_configuration
+
+
+def _extract_version(package_name):
+    try:
+        return pkg_resources.get_distribution(package_name).version
```

Comment:
Does the setup.cfg file get installed alongside the module? I assume not and if that's the case I think it'd be preferable to flip the order of things here, e.g. look for setup.cfg first and then fall back to the pkg_resources based version. 

It'd be pretty edge-case-y, but it seems like it'd be possible to have a system or pip installed version that isn't the one loaded into the python interpreter in which case you'd have code that doesn't match the installed version.